package.json

```diff
@@ -57,4 +57,4 @@
     "truffle": "^5.1.26",
     "truffle-hdwallet-provider": "^1.0.0-web3one.1"
   }
-}
+}
```

Comment:
nit: missing newline at the end of file.

Reply:
```suggestion
}

```